Source/Rhetos.Utilities.Test/LegacyUtilitiesTests.cs

```diff
@@ -67,5 +67,25 @@ public void PathThrowsOnNullEnvironment()
             Paths.Initialize(null, null, null, new AssetsOptions());
             TestUtility.ShouldFail<FrameworkException>(() => Console.WriteLine(Paths.GeneratedFolder), "AssetsFolder expected to be configured");
         }
+
+        [TestMethod]
+        public void PathsInitializationTest()
+        {
+            var rootPath = @"C:\RhetosServer";

```

Comment:
This test might fail in future versions because it is base on invalid Rhetos app folder.
Please move this test to integration tests (CommonConceptsTest.sln), so that it can be executed on a valid folder with live Rhetos application.